src/lib/components/FormModel/children/Form/UI/DateTimePicker/date-time-picker.component.js

```diff
@@ -115,11 +115,11 @@ const DateTimePicker = React.memo(
        browser format samples: 'es', 'en-US', 'en-GB'
        date-fns format samples: 'es', 'enUS', 'enGB' */
     let locale = getLocale().split('-');
-    if (locale[2]) {
-      locale[2] = locale[2].toUpperCase();
-      locale = `${locale[0]}${locale[2]}`;
+    if (locale[1]) {
+      locale[1] = locale[1].toUpperCase();
+      locale = `${locale[0]}${locale[1]}`;
     } else {
-      locale = `${locale[0]}`;
+      locale = `${locale[1]}`;
```

Comment:
Not sure I follow this last bit - if there is no locale[1], then set locale to locale[1]?